pom.xml

```diff
@@ -50,7 +50,37 @@
         <groupId>org.sonatype.plugins</groupId>
         <artifactId>nexus-staging-maven-plugin</artifactId>
       </plugin>
+      <plugin>
```

Comment:
You've also moved the nexus-staging-maven-plugin from pluginManagement to plugins, it needs to be moved back

Reply:
Sad face